metal/config.go

```diff
@@ -114,10 +113,6 @@ func getMetalConfig(providerConfig io.Reader) (Config, error) {
 	if loadBalancerSetting != "" {
 		config.LoadBalancerSetting = loadBalancerSetting
 	}
-	// and set for default
-	if config.LoadBalancerSetting == "" {
-		config.LoadBalancerSetting = defaultLoadBalancerConfigMap
-	}
```

Comment:
As far as I can tell this didn't actually work as a default, since `metallb-system:config` does not parse as a URL.

Reply:
Again, good catch. It didn't break anything, as the part that processes the URL only sets the configmap if it is a valid URL.